### ISSUE TITLE
Notification conflict between mex ones and GNOME ones

### ISSUE THREAD

**Issue by dlespiau** [media-explorer/media-explorer#82]
We have notifications at the top of the application. GNOME has its own notification system. Just switching to use them breaks quite a bit the design. What to do then? Need to experiment.

**Comment by hadess** [comment]
What sort of notifications are there? Got screenies?

**Comment by dlespiau** [comment]
http://damien.lespiau.name/files/temp/mex-notification.png

This one is probably fine, but we also have system level notifications like USB disk insertion/removal and nework going online/offline there.

**Comment by hadess** [comment]
Right, those probably need to be completely disabled for the GNOME mode. There would already be a popup for removable media: http://blogs.gnome.org/cosimoc/2011/06/23/hotplug-hotness/ and notifications for network disconnection.

**Comment by semiot** [comment]
Hi, 

So I've spent a little time mocking up what MEX might look like if we were to integrate it with the GNOME UI more closely. 

As our primary use case for MEX remains a lean-back experience where you're more likely to have a remote control instead of a mouse, there may be some conflicts between the standard desktop interaction and the lean-back experience. I'm most concerned about the message tray including actionable items like "Open with Documents" and the potential to occlude controls in MEX. 

I've added the mockups to the GNOME wiki page hadess created a few days ago:
https://live.gnome.org/Design/Proposals/FullscreenControls

I've also laid out two possible solutions after the screenshots. It might make sense to put these both in code just to get a better sense of how this all feels, though my hunch is that the GNOME message tray may still be in a bit of conflict the rest of MEX.
